src/test/java/api/support/builders/FeeFineBuilder.java

```diff
@@ -0,0 +1,76 @@
+package api.support.builders;
+
+import static org.folio.circulation.support.JsonPropertyWriter.write;
+
+import java.util.UUID;
+
+import io.vertx.core.json.JsonObject;
+
+public class FeeFineBuilder extends JsonBuilder implements Builder {
+  private UUID id;
+  private UUID ownerId;
+  private String feeFineType;
+  private Double defaultAmount;
+  private String chargeNoticeId;
+  private String actionNoticeId;
+  private JsonObject metadata;
```

Comment:
Why does the builder need to define the change metadata?

Reply:
Removed metadata field. I guess I just added all the fields from JSON.